common/core/network.go

```diff
@@ -221,6 +226,48 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) (resultError error) {
 	return nil
 }
 
+func addEndpointGatewaysFromConfig(
+	endpointInfo *network.EndpointInfo,
+	cniConfig *cni.NetworkConfig) {
+
+	defaultDestipv4, defaultDestipv4Network, _ := net.ParseCIDR("0.0.0.0/0")
+	defaultDestipv6, defaultDestipv6Network, _ := net.ParseCIDR("::/0")
+
+	for _, addr := range cniConfig.AdditionalRoutes {
+
+		var isv4 bool
+		if addr.GW.To4() != nil {
+			isv4 = true
+		}
+
+		if isv4 {
+			if endpointInfo.Gateway == nil {
+			   m1, _ := addr.Dst.Mask.Size()
```

Comment:
nit: looks like there is a mix of tabs and spaces in this block

Reply:
Fixed.